src/components/Checkout/Steps.tsx

```diff
@@ -0,0 +1,31 @@
+type StepsProps = {
+  step: number;
+  setStep: (step: number) => void;
+};
+
+const Steps = ({ step, setStep }: StepsProps) => {
+  return (
+    <div className="checkout-steps">
+      <div
+        className={`checkout-step ${step === 1 ? "active" : ""}`}
```

Comment:
This will work perfectly, however it's pretty hard coded. I would personally create an enum with the checkout steps. Set these enum values in de useState and check on the enums, this will prevent hardcoded values like 1,2,3 and will be more clear to read the code